server/docker/Dockerfile.base-env

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:20.04
+FROM ubuntu:22.04
```

Comment:
I love it!
I've wanted to update that so many times, but the scope was always so large.
With this setup, if feels so much easier to make it all work.
Well done!